guides/RELEASE-PROCESS.md

```diff
@@ -12,13 +12,33 @@
 
       **Note**: Your NPM account must access to the CMS group and provide a valid access token. To add a token, edit your `~/.npmrc` file so the contents are `//registry.npmjs.org/:_authToken={token}`
 
-   1. Run the release script to automatically begin the release process and publish to NPM.
+   1. Run the prepublish script to automatically prepare the NPM package for publishing.
+
       ```
-      yarn release
+      yarn pre-release
       ```
+
       This script will install dependencies, build design system assets, and run the linter and tests. If everything passes, `lerna` will prompt you for the new version number, which should follow the [SemVer release format](#versioning). After the versions are updated, the script will create a tagged release commit (prefaced with `core-`), a branch (prefaced with `release-`) containing the release commit, and a zip for the Github release notes.
-   1. The last step in the script will prompt you to continue with publishing to NPM. Proceed if everything is ready.
-   1. After the release script is completed, make sure to merge the release branch (i.e. `release-2.0.0`) into master.
+
+   1. After the prepublish script is completed, visually check that release zip created from the release script (i.e. `cmsgov-design-system-2.0.0.tgz`) contains the relevant folders and files
+
+      - cmsgov-design-system includes `dist`, `src` folders and `package.json`, `README.md` files
+      - cmsgov-design-system-docs includes `src` folder and `package.json`, `README.md` files
+      - cmsgov-design-system-scrits includes `gulp`, `jest` folders and `cli.js`, `configDefaults.js`, `package.json`, `README.md` files
+
+   1. The next step is to run the lerna publish command to publish to NPM.
+
+      ```
+      yarn release
+      ```
+
+   1. After the publish process is completed, check [NPM](https://www.npmjs.com/package/@cmsgov/design-system) to verify the package is updated.
```

Comment:
I dont think this is necessary, as the publish script will tell you if its failed or not